### PR TITLE
Chaos Labs Supply and Borrow Cap Updates - Aave V3 

### PR DESCRIPTION
**Pull request by Maltmark**
Updates Supply and Borrow Cap for Optimism, Arbitruma and polygon.

https://governance.aave.com/t/arc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-2023-02-24/12048

### CODE REVIEW

File: src/contracts/optimism/AaveV3OptCapsPayload-Feb26.sol [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    AaveV3Optimism.POOL_CONFIGURATOR.setBorrowCap(SUSD, SUSD_BORROW_CAP);
+
+    AaveV3Optimism.POOL_CONFIGURATOR.setSupplyCap(USDC, USDC_SUPPLY_CAP);
+    AaveV3Optimism.POOL_CONFIGURATOR.setBorrowCap(USDC, USDC_BORROW_CAP);
+    
+    AaveV3Optimism.POOL_CONFIGURATOR.setSupplyCap(USDT, USDT_SUPPLY_CAP);
+    AaveV3Optimism.POOL_CONFIGURATOR.setBorrowCap(USDT, USDT_BORROW_CAP);
+
+    AaveV3Optimism.POOL_CONFIGURATOR.setSupplyCap(AAVE, USDT_SUPPLY_CAP);
```
Comment (sakulstra) [inline src/contracts/optimism/AaveV3OptCapsPayload-Feb26.sol:54]: aave setting usdt supply cap :warning:

Comment (Maltmark) [reply src/contracts/optimism/AaveV3OptCapsPayload-Feb26.sol]: Thank you! great catch. I updated the payload and the test.

File: src/contracts/arbitrum/AaveV3ArbBorrowCapsPayload-Feb26.sol [hunk 2]
```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {AaveV3Arbitrum, AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IPoolConfigurator, ConfiguratorInputTypes} from 'aave-address-book/AaveV3.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @dev This payload sets borrow caps for multiple assets on AAVE V3 Arbitrum
+ * - Snapshot: TBD //TODO
```
Comment (sakulstra) [inline src/contracts/arbitrum/AaveV3ArbBorrowCapsPayload-Feb26.sol:10]: remove the todo when not planning todo

File: src/contracts/optimism/AaveV3OptCapsPayload-Feb26.sol [hunk 3]
```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {AaveV3Optimism, AaveV3OptimismAssets} from 'aave-address-book/AaveV3Optimism.sol';
+import {IPoolConfigurator, ConfiguratorInputTypes} from 'aave-address-book/AaveV3.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @dev This payload sets borrow caps for multiple assets on AAVE V3 Optimism
+ * - Snapshot: TBD //TODO
```
Comment (sakulstra) [inline src/contracts/optimism/AaveV3OptCapsPayload-Feb26.sol:10]: same here etc

File: src/contracts/arbitrum/AaveV3ArbBorrowCapsPayload-Feb26.sol (hunk 2 above)
Comment (Maltmark) [reply src/contracts/arbitrum/AaveV3ArbBorrowCapsPayload-Feb26.sol]: removed

**Review by sakulstra — APPROVED** [review]

**Pull request merged**